decompiler/data/game_text.cpp

```diff
@@ -150,6 +151,11 @@ std::string write_game_text(
   }
   std::sort(languages.begin(), languages.end());
 
+  // hack
+  if (version == GameTextVersion::JAK1_V1) {
```

Comment:
is this supposed to be here?

Reply:
yeah, but now that I think about it I can probably change the black label config's `text_version` to be the same as PAL+ since we apply a patch now and avoid the hack here altogether.